cpp/test/Ice/properties/Client.cpp

```diff
@@ -111,6 +111,45 @@ Client::run(int, char**)
         }
         cout << "ok" << endl;
     }
+
+    {
+        cout << "testing ice properties with set default values... " << flush;
+        Ice::PropertiesPtr properties = Ice::createProperties();
+        string toStringMode = properties->getIceProperty("Ice.ToStringMode");
+        test(toStringMode == "Unicode");
+        int32_t closeTimeout = properties->getIcePropertyAsInt("Ice.Connection.CloseTimeout");
+        test(closeTimeout == 10);
+        vector<string> retryIntervals = properties->getIcePropertyAsList("Ice.RetryIntervals");
+        test(retryIntervals.size() == 1);
+        test(retryIntervals[0] == "0");
+        cout << "ok" << endl;
+    }
```

Comment:
To be complete we should also test that you don't get the default when the property is set to a different value. But maybe not worth because the code is simple enough?

Reply:
I don't think it's needed. The rest of the tests will cover this when we start using it.